packages/techdocs-cli/src/lib/utility.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  RemoteProtocol,
+  ParsedLocationAnnotation
+} from "@backstage/techdocs-common";
+import * as winston from "winston";
+
+export const convertTechDocsRefToLocationAnnotation = (
+  techdocsRef: string
+): ParsedLocationAnnotation => {
+  // Split on the first colon for the protocol and the rest after the first split
+  // is the location.
+  const [type, target] = techdocsRef.split(/:(.+)/) as [
+    RemoteProtocol?,
+    string?
+  ];
+
+  if (!type || !target) {
+    throw new Error(
+      `Can not parse --techdocs-ref ${techdocsRef}. Should be of type HOST:URL.`
+    );
+  }
+
+  return { type, target };
+};
+
+export const createLogger = ({
+  verbose = false
```

Comment:
What do you think of adding a `prefix` argument here and handling the log prefixing in the transport formatter below?  [e.g.](https://stackoverflow.com/questions/42858585/add-module-name-in-winston-log-entries)

Could make it a little easier to scan some of the code in the command methods above.

Reply:
The logs are already prefixed by their level i.e. `info` or `verbose`, if that is what you meant. 😄 

```
backstage/techdocs-container/mock-docs  main ✔                                           49d22h
▶ techdocs-cli serve -v
info: Starting mkdocs server.
verbose: [docker/mkdocs] INFO    -  Building documentation...
info: Waiting for mkdocs server to start...
verbose: [docker/mkdocs] WARNING -  Config value: 'dev_addr'. Warning: The use of the IP address '0.0.0.0' suggests a production environment or the use of a proxy to connect to the MkDocs server. However, the MkDocs' server is intended for local development purposes only. Please use a third party production-ready server instead.
verbose: [docker/mkdocs] INFO    -  Cleaning site directory
verbose: [docker/mkdocs] DEBUG   -  Successfully imported extension module "plantuml_markdown".
verbose: [docker/mkdocs] DEBUG   -  Successfully loaded extension "plantuml_markdown.PlantUMLMarkdownExtension".
info: Waiting for mkdocs server to start...
verbose: [docker/mkdocs] /usr/local/lib/python3.8/site-packages/pymdownx/superfences.py:662: PymdownxDeprecationWarning:
verbose: [docker/mkdocs] The tab option in SuperFences has been deprecated in favor of the general purpose 'pymdownx.tabbed' extension.
verbose: [docker/mkdocs] While you can continue to use this feature for now, it will be removed in the future.
verbose: [docker/mkdocs] Also be mindful of the class changes, if you require old style classes, please enable the 'legacy_tab_classes' option.
verbose: [docker/mkdocs]   warnings.warn(MSG_TAB_DEPRECATION, PymdownxDeprecationWarning)
verbose: [docker/mkdocs] DEBUG   -  Successfully loaded extension "mkdocs.structure.pages._RelativePathExtension".
....
```

Reply:
Oh, I was just seeing a lot of `logger.verbose(`${logPrefix} ${line}`);` and some similar, associated boilerplate logging code throughout this changeset and thought perhaps there was opportunity to centrally manage that kind of thing.  But perhaps you want to differentiate within a single context what log line is coming from what?  Either way, not a huge thing.

Reply:
Right. The `logPrefix` is either `[docker/mkdocs]` or `[mkdocs]`. It is only useful for the `serve` and `serve:mkdocs` command as of now. I think it can also be used in `generate` and `publish` commands. Will think about it in future iterations. :)